src/custom-errors/ApiCallError.ts

```diff
@@ -0,0 +1,6 @@
+export class ApiCallError extends Error {
```

Comment:
I think we can just include this in `@tutorbook/model/api.ts`. Is there a reason it shouldn't be @JKorum?